otter/integration/lib/autoscale.py

```diff
@@ -299,7 +301,9 @@ def decide(resp):
             if resp.code == 200:
                 return self.treq.json_content(resp).addCallback(
                     lambda x: (200, x))
-            return (404, None)
+
+            return self.treq.content(resp).addCallback(
```

Comment:
Reviewer: the reasons for this change are:

1.  was a previously a bug, since it would return 404 no matter what, and the success code was passable (sorry, I forgot about this one when adding success codes)
2.  twisted can sometimes have a hanging issue if we don't read the response from a request and we use the same connection over and over again.  Not as much of an issue here because the tests kill the pool in between every test, but just in case...